logfire/_internal/config.py

```diff
@@ -904,7 +907,7 @@ def add_span_processor(span_processor: SpanProcessor) -> None:
                         self.token = credentials.token
                         self.advanced.base_url = self.advanced.base_url or credentials.logfire_api_url
 
-                if self.token is not None:
+                if is_valid_token(self.token):
```

Comment:
```suggestion
                if self.token:
```

then the assert shouldn't be needed

Reply:
yea,  earlier I thought abt handling `''` token from cred file also, it doesn't make sense.
committing